src/reducers-actions/orders.ts

```diff
@@ -32,15 +32,31 @@ export const INITIAL_ORDERS_STATE = { orders: [], offset: 0 }
 export const reducer = (state: OrdersState, action: ReducerActionType): OrdersState => {
   switch (action.type) {
     case 'UPDATE_OFFSET':
-    case 'OVERWRITE_ORDERS': {
       return { ...state, ...action.payload }
+    case 'OVERWRITE_ORDERS': {
+      const {
+        payload: { orders },
+      } = action
+
+      // default sorting order is ascending by order id
+      // since we are starting from 0, reverse to have the latest on top
+      orders.reverse()
```

Comment:
Is it ok to mutate the array here? I guess it's not used anywhere else and only dispatched right after creation, but if it wasn't, mutating would be a bad idea.

Reply:
YOUVE ACTIVATED MY TRAP CARD

Reply:
I'll merge this for the user test, I think we can decide later what to do with this. My initial thought is that is fine, although if there's a risk, we can always create a new instance of the array before reversing.

If we do this, let's do it in another PR

Reply:
https://github.com/gnosis/dex-react/pull/870